server/app/_components/TodoForm.tsx

```diff
@@ -41,8 +41,10 @@ const TodoForm = ({ children }): JSX.Element => {
 
   return (
     <>
-      { todoList && todoList.map((item) => (
-        <TodoItem key={item.id} todo={item} onEditBeginingHandler={onTodoEditBegining} />
+      { todoList && todoList.map((item, index) => (
+        <TodoItem key={item.id} todo={item} 
+        isActive={index === editingTodoIndex}
+        onEditBeginingHandler={onTodoEditBegining} />
```

Comment:
山括弧で囲んだ部分の範囲が、どこからどこまでなのかが分かりにくいように思いました。
括弧内の要素は次のようにインデントを付けることができ、こちらの方が理解しやすくなると思います。
```suggestion
        <
          TodoItem key={item.id} todo={item} 
          isActive={index === editingTodoIndex}
          onEditBeginingHandler={onTodoEditBegining}
        />
```